source/administration/changelog.md

```diff
@@ -4,6 +4,123 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Users can can also use these bots to post to any channel in the system, whether it’s a private team, private channel or a direct message channel.
+ - Integrations tied to bot accounts (instead of user accounts) no longer break if a user leaves the company and their account gets deactivated.
+ 
+#### Jira V2.0
+ - Deep two-way integration between Jira and Mattermost.
+ - Users can send issue creation, issue updates and comments to Mattermost channels. Users can also take quick actions in Mattermost, including creating Jira issues, attaching Mattermost messages to Jira issues and transitioning issues via slash commands.
+ - For a full feature set for 2.0, see https://github.com/mattermost/mattermost-plugin-jira#jira-20-features.
+
+#### System Console Reorganization
+ - Informational architecture restructure of the system console to make a more logical flow to the settings and to provide a more cohesive experience for hiding features on the Mattermost Private Cloud product, where the system admin should not have access to change configurations that affect the environment directly.
+
+#### Net Promoter Score (NPS)
+ - We are gathering user feedback that we will use to improve the user experience and hear directly from our users. The feature can be disabled via **XXXXXX**.
+ 
+#### LDAP Group Sync Removals
+ - Users can manage the membership of private teams and channels with groups, eliminating the need to individually add and remove members. Users in the groups will be automatically removed from the team or channel when removed from an associated group.
+ 
+#### User/Channel Search & AutoComplete in ElasticSearch
+ - Added a new flag in the **System console > Elasticsearch** to enable the usage of Elasticsearch for autocompletion queries. When this flag is enabled, Elasticsearch will use its indexed data for the autocompletions.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added an option to add a user to a channel from the profile pop-over.
+ - Removed ``@`` for full name display in push notifications.
+ 
+#### Plugins
+ - Added support for markdown in plugin system console help text fields.
+ - Added support for plugins to override ephemeral posts.
+
+#### Localization
+ - Promoted Polish language to "official".
+
+#### Command Line Interface (CLI)
+ - Added a ``command modify`` CLI command to be able to modify slash commands.
+ - Added support for converting user accounts to bot accounts through the CLI.
+ - Implemented a new command ``MigrateConfig`` for migrating configuration to and from the database.
+ - For LDAP Groups, added the ability to switch a team or channel to be group-constrained via the CLI.
+ 
+#### Administration
+ - Added support for running two Mattermost instances on the same domain using subpaths.
+ - Added support for importing threads from Slack.
```

Comment:
what about the corporate Slack import?

Reply:
The PR hasn't been merged yet. Should I add it already?

Reply:
Ah, didn't realize it wasn't merged yet. I guess it's gonna be too late for 5.12. Are we doing a custom build for the customer instead?